### PR TITLE
feat: Add theme page

### PR DESCRIPTION
**Pull request by KeithClinard**

### CODE REVIEW

File: apps/docs-app/app/components/theme-page.gts [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import Button from '@nrg-ui/core/components/button';
+import Footer from '@nrg-ui/core/components/footer';
+import Form from '@nrg-ui/core/components/form';
+import Icon from '@nrg-ui/core/components/icon';
+import CardContainer from '@nrg-ui/core/components/mktg/card-container';
+import Header from '@nrg-ui/core/components/mktg/header';
+import SectionHeader from '@nrg-ui/core/components/mktg/section-header';
+import bind from '@nrg-ui/core/helpers/bind';
```
Comment (TSenter) [inline apps/docs-app/app/components/theme-page.gts:11]: All of these can be combined into a single import
```suggestion
import {
  bind,
  Button,
  Footer,
  Form,
  Icon,
  MktgCardContainer,
  MktgHeader,
  MktgSectionHeader
} from '@nrg-ui/core';
```

File: apps/docs-app/app/components/theme-page.gts [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+  phone = '';
+
+  @tracked
+  radio = '';
+
+  @tracked
+  checkbox;
+
+  @tracked
+  checkboxGroup = autoTrack(new Array(3));
```
Comment (TSenter) [inline apps/docs-app/app/components/theme-page.gts:43]: Can this just be specified as a tracked array? The import would be simpler.
```suggestion
  checkboxGroup = new TrackedArray(3);
```

**Review by TSenter — CHANGES_REQUESTED** [review]

File: apps/docs-app/app/components/theme-page.gts (hunk 2 above)
Comment (KeithClinard) [reply apps/docs-app/app/components/theme-page.gts:45]: Tried that. Got the following error message:
image

Seems using the `autoTrack()` method is how we got around this problem in the forms demo page.

File: apps/docs-app/app/components/theme-page.gts (hunk 1 above)
Comment (TSenter) [reply apps/docs-app/app/components/theme-page.gts]: I didn't comment on this before, but your `<template>` will need to be updated. `CardContainer` -> `MktgCardContainer`, `Header` -> `MktgHeader`, etc. This also needs a trailing comma (linting).

**Review by TSenter — APPROVED** [review]

**Pull request merged**